docs/docsite/rst/vault.rst

```diff
@@ -3,7 +3,7 @@ Ansible Vault
 
 .. contents:: Topics
 
-New in Ansible 1.5, "Vault" is a feature of ansible that allows keeping sensitive data such as passwords or keys in encrypted files, rather than as plaintext in your playbooks or roles. These vault files can then be distributed or placed in source control.
+The "Vault" is a feature of ansible that allows you to keep sensitive data such as passwords or keys in encrypted files, rather than as plaintext in playbooks or roles. These vault files can then be distributed or placed in source control.
```

Comment:
We don't use ```The``` with vault.  If it seem funny to just say vault,  perhaps Ansible Vault or "ansible-vault" (after the command line tool)?

Reply:
Got it.